hpx/runtime/threads/policies/lockfree_queue_backends.hpp

```diff
@@ -11,6 +11,7 @@
 #include <hpx/config.hpp>
 
 #include <hpx/util/lockfree/deque.hpp>
+#include <hpx/concurrent/concurrentqueue.h>
```

Comment:
Why not `hpx/util/lockfree/concurrentqueue.h` like the deque?

Reply:
When we were talking about integrating libcds with hpx, we had the idea of making hpx/concurrent as a new place for concurrent stuff. But I'm quite happy to put it into `util/lockfree/` since its a single header

Reply:
Or move deque into concurrent?

Reply:
Either works for me

Reply:
I don't mind. The 'concurrent/concurrentqueue.h' has lots of redundant information. Also, please use the `.hpp` file ending to stay consinstent with the rest.

Reply:
I've renamed the file to `.hpp` extension and moved it to hpx/util/lockfree/ - force pushed with the commits edited accordingly.